src/command_line_parser.cc

```diff
@@ -507,6 +508,10 @@ std::vector<Option> TritonParser::recognized_options_
       {OPTION_METRICS_INTERVAL_MS, "metrics-interval-ms", Option::ArgFloat,
        "Metrics will be collected once every <metrics-interval-ms> "
        "milliseconds. Default is 2000 milliseconds."},
+      {OPTION_METRICS_CONFIG, "metrics-config", "<string>=<string>",
+       "Specify a metrics-specific configuration setting. The format of this "
+       "flag is --metrics-config=<setting>=<value>. It can be specified "
```

Comment:
How will the CLI setting look going forward? Will `--metrics-config=<setting>=<value>` set global configs when the name-specific metrics are enabled? I want to make sure the API stays consistent in this case.

Reply:
Same as backend-config and cache-config:
- `--metrics-config setting=value` will set global configs for `name=""` (only supported format currently)
- `--metrics-config name,setting=value` will set name-specific configs for `name=name` if eventually added back in, but currently is rejected by CLI